packages/openapi-code-generator/src/typescript/common/schema-builders/zod-schema-builder.ts

```diff
@@ -133,6 +133,15 @@ export class ZodBuilder extends AbstractSchemaBuilder<
       .join(".")
   }
 
+  protected transform(
+    schema: string,
+    transformation: string | ((it: unknown) => unknown),
+  ) {
+    return [schema, `transform(${transformation.toString()})`]
```

Comment:
We probably actually need to use a `preprocess` here to get the desired effect, eg:
```
> z.preprocess(it => Array.isArray(it) ? it : [it], z.array(z.number())).parse([1,2])
[ 1, 2 ]
> z.preprocess(it => Array.isArray(it) ? it : [it], z.array(z.number())).parse(1)
[ 1 ]
```